app/models/recipe_food.rb

```diff
@@ -0,0 +1,4 @@
+class RecipeFood < ApplicationRecord
+  belongs_to :foods
+  belongs_to :recipes
+end
```

Comment:
It is better to use the rails naming convention for defining belongs to references, like using the singular model name for belongs to reference. Like 
    belongs_to :food
  belongs_to :recipe